shoop/simple_pricing/admin_form_part.py

```diff
@@ -69,7 +69,7 @@ def _process_single_save(self, shop, group):
         if clear:
             SimpleProductPrice.objects.filter(product=self.product, group=group, shop=shop).delete()
         else:
-            spp, _ = SimpleProductPrice.objects.get_or_create(product=self.product, group=group, shop=shop)
+            spp, _ = SimpleProductPrice.objects.get_or_create(product=self.product, group=group, shop=shop, price=value)
             spp.price = value
             spp.save()
```

Comment:
Good points. :) This commit would probably actually break things more than fix them -- which makes it curious that Jenkins thinks everything is hunky dory.